packages/nc-gui-v2/components/general/SocialCard.vue

```diff
@@ -22,16 +76,16 @@ const isRtlLang = $computed(() => ['fa'].includes(currentLang.value))
             </div>
             <div v-else class="flex items-center">
               <!-- Star -->
-              {{ $t('labels.community.starUs1') }}
-              <mdi-star-outline class="mx-1" />
+              {{ `${$t('labels.community.starUs1')} ` }}
```

Comment:
Why the double string literal? If you need to add whitespace you can just use `&nbsp;`

Reply:
somehow adding `&nbsp;` added more space, will try to update

Reply:
any additional whitespace we have to remove, will become like

```
{{ $t('labels.community.starUs1')}}&nbsp;<!-- us on Github -->{{ $t('labels.community.starUs2') }}
```

Reply:
Following seems fine 

```
              <!-- Star -->
              {{ $t('labels.community.starUs1') }}
              {{ ' ' }}
              <!-- us on Github -->
              {{ $t('labels.community.starUs2') }}
```

Reply:
Because you're doing a double string literal and one of them has whitespace in it.
You're using {{ `${$t(...)} ` <--------- this }}